draft-ietf-webtrans-http2.md

```diff
@@ -284,19 +284,49 @@ WebTransport streams and datagrams that belong to different WebTransport
 sessions are identified by the CONNECT stream on which they are transmitted,
 with one WebTransport session consuming one CONNECT stream.
 
-## Transport Considerations
+## Transport Properties
+
+The WebTransport framework [OVERVIEW] defines a set of optional transport
```

Comment:
```suggestion
The WebTransport framework {{OVERVIEW}} defines a set of optional transport
```
This is more robust, as `[OVERVIEW]` will render even if there is no reference by that name.